files/bootstrap.sh

```diff
@@ -392,6 +399,22 @@ Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
 EOF
 fi
 
+# Replace dockerd with containerd
+if [[ "$CONTAINER_RUNTIME" -eq "containerd" ]]; then
+    cat <<EOF > /etc/systemd/system/kubelet.service.d/50-kubelet-container-runtime.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/kubelet --cloud-provider aws \
+    --config /etc/kubernetes/kubelet/kubelet-config.json \
+    --kubeconfig /var/lib/kubelet/kubeconfig \
+    --container-runtime remote \
+    --container-runtime-endpoint "unix:///run/containerd/containerd.sock" \
+    --network-plugin cni $KUBELET_ARGS $KUBELET_EXTRA_ARGS
+EOF
+elif [[ "$CONTAINER_RUNTIME" -ne "dockerd" ]]; then
```

Comment:
Same as before, we should be using `=` not `-ne` because this is for numeric comparison and will fail.